portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/antisamy/AntiSamy.testcase

```diff
@@ -176,4 +217,78 @@ definition {
 		}
 	}
 
+	@description = "LPD-25553: Assert AntiSamy is correctly disabled"
+	@priority = 3
+	test ScriptInWikiPortletShouldNotBeSanitizedWhenAntiSamyIsDisabled {
+		property antisamy.enabled = "true";
```

Comment:
Could we just change this value to `false` and remove the "Disable AntiSamy" task?